connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerConfigTransformerTest.java

```diff
@@ -68,19 +73,25 @@ public void testReplaceVariable() {
 
     @Test
     public void testReplaceVariableWithTTL() {
+        // Raise error if connectorConfigReloadAction is called, since it results in infinite recursion
+        EasyMock.expect(herder.connectorConfigReloadAction(MY_CONNECTOR)).andThrow(new AssertionFailedError()).anyTimes();
```

Comment:
These aren't needed in these tests anymore, and `anyTimes()` wouldn't be ok here anyway. I think the `anyTimes()` was masking the fact that we could remove these now.